src/components/commons/inputs/Dropdown.tsx

```diff
@@ -55,12 +53,12 @@ export const Dropdown = ({
         <select
           className={cx('select-group-select')}
           {...register(name, {
-            value: typeof options[0] === 'object' ? options[0].id : options[0],
+            value: options[0].value,
           })}
         >
           {options.map((option, index) => (
-            <option key={`dropdown-${index}`} value={typeof option === 'object' ? option.id : option}>
-              {typeof option === 'object' ? option.id : option}
+            <option key={`dropdown-${index}`} value={option.value}>
+              {option?.title}
```

Comment:
map 메서드로 요소가 반환되었다는 건, option이 존재한다는 거니까 물음표는 없어도 될거같습니다!
```suggestion
              {option.title}
```